pom_confluent.xml

```diff
@@ -49,6 +49,9 @@
         <skipTests>false</skipTests>
         <skipUnitTests>${skipTests}</skipUnitTests>
         <skipIntegrationTests>${skipTests}</skipIntegrationTests>
+        <license.processing.dependencyJarsDir>${project.build.directory}/dependency-jars</license.processing.dependencyJarsDir>
+        <license.processing.dependencyListFile>${project.build.directory}/dependency-list.txt</license.processing.dependencyListFile>
+        <license.processing.targetDir>${project.build.directory}/../licenses</license.processing.targetDir>
```

Comment:
I prefer if you not pollute the main directory with generates sources. Target directory in maven is specifically for that. 
Also, if you run `clean` the `licenses` directory is not cleaned up.

Reply:
Added cleanup for licenses directory